packages/wrangler/e2e/helpers/normalize.ts

```diff
@@ -167,3 +168,7 @@ function removeStandardPricingWarning(stdout: string): string {
 		""
 	);
 }
+
+function removeZeroWidthSpaces(stdout: string) {
+	return stdout.replaceAll(/\u200a|\u200b/g, " ");
```

Comment:
It feels like an implementation detail. The zero-width space is for clack to not remove the indentation spacing. Doesn't need to be in the snapshots – since the absence of the zwsp will mean no indentation in the snapshot which will be caught and fail the test